deploy/kube-templates/glusterblock-provisioner.yaml

```diff
@@ -0,0 +1,28 @@
+---
+kind: List
+apiVersion: v1
+items:
+- kind: Deployment
+  apiVersion: extensions/v1beta1
+  metadata:
+    name: glusterblock-provisioner
+    labels:
+      glusterfs: glusterblock-provisioner-deployment
+    annotations:
+      description: Defines how to deploy the glusterblock provisioner pod.
+  spec:
+    replicas: 1
+    template:
+      metadata:
+        name: glusterblock-provisioner
+        labels:
+          glusterfs: glusterblock-provisioner-pod
+      spec:
+        serviceAccountName: glusterblock-provisioner
+        containers:
+        - name: glusterblock-provisioner
+          image: gluster/glusterblock-provisioner:latest
+          imagePullPolicy: IfNotPresent
+          env:
+          - name: PROVISIONER_NAME
+            value: glusterblock-provisioner
```

Comment:
Yeah, I had it changed in the template to `gluster.org/glusterblock` , however failed to update you in the mail. so appreciated if you can keep this here as well.